_locales/en/messages.json

```diff
@@ -1182,10 +1182,43 @@
     "description": "Text for link in toolbar pop-up to write a new style for the current URL"
   },
   "bckpInstStyles": {
-      "message": "Export styles"
+    "message": "Export styles"
   },
   "retrieveBckp": {
-      "message": "Import styles"
+    "message": "Import styles"
+  },
+  "bckpDropboxStyles": {
```

Comment:
whoever added `bckp` in our messages was wrong so let's not repeat the mistake: please use `backup` in the new messages. I also think it might be a good idea to prefix all the new messages with something like uhm... `sync` maybe?